templates/engage/_base_engage_markdown.html

```diff
@@ -31,7 +31,7 @@
   <div class="row u-equal-height">
     <div class="{% if header_image %}col-7 {% else %}col-8{% endif %} u-vertically-center">
       <div>
-        <h1 {% if not header_subtitle and not header_date %}class="u-no-margin--bottom"{% endif %}>
+        <h1 {% if not header_subtitle and not header_date %}class="u-no-margin--bottom{% if title_class %} {{ title_class }}{% endif %}"{% endif %}>
```

Comment:
Shouldn't first `{% if %}` wrap only `u-no-margin--bottom`, not whole class name?
It seems like these are 2 independent conditions.

Reply:
Oh you're right - going to take a look and see if I can make this just one conditional.